vspace/src/main/java/edu/asu/diging/vspace/web/staff/AddExternalLinkController.java

```diff
@@ -33,7 +33,7 @@
         if (space == null) {
             return new ResponseEntity<>("{'error': 'Space could not be found.'}", HttpStatus.NOT_FOUND);
         }
-        IExternalLinkDisplay display = spaceManager.createExternalLink("test", space, new Float(x), new Float(y), url);
+        IExternalLinkDisplay display = spaceManager.createExternalLink(url, space, new Float(x), new Float(y));
```

Comment:
we do need a title for the external link

Reply:
The url itself is given as title for createExternalLink method. Wouldn't that work?

Reply:
no, there should be a label (e.g. if the user wants to linnk to Google, they might want to say something like "click here to go to Google" and not "http://google.com/".